build

```diff
@@ -31,10 +32,16 @@ os.environ['COPYFILE_DISABLE'] = 'true'
 USER = 'nuodb'
 GROUP = 'nuodb'
 
+
+def is_macos():
+    return platform.system() == "Darwin"
+
+
 def build_clients(packages):
-    version = datetime.now().strftime("%Y%m%d")
+    version = Globals.version
+    build_number = Globals.build_number
     target = Globals.target
-    pkgname = 'nuodb-client-{}.{}'.format(version, target)
+    pkgname = 'nuodb-client-{}-{}.{}'.format(version, build_number, target)
```

Comment:
We're changing the format of the package name... is this something everyone is aware of and is OK with?